assets/css/global.scss

```diff
@@ -0,0 +1,264 @@
+// CSS RESET
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Open Sans", sans-serif;
+  color: white;
+  // outline: 1px solid white; //uncomment this line to diagnose position issues
+}
+
+$accent-color: #399fd3;
+
+body {
+  background-color: #222222;
+  color: white;
+  font-family: Helvetica;
+  font-size: 100%;
+}
+
+.small-hero {
+  height: 350px;
```

Comment:
Can you make the height of this closer to 500px? At 350px, the picture of the Monument on the Volunteer page is cropped and it's hard to see the mosaics on the monument.

Reply:
Set to 500px. May need further adjustment for mobile layouts.